luigi/task.py

```diff
@@ -289,6 +289,9 @@ def __init__(self, *args, **kwargs):
         self.task_id = task_id_str(self.task_family, self.to_str_params(only_significant=True))
         self.__hash = hash(self.task_id)
 
+        self.status_message = None
+        self._status_message_callback = None
```

Comment:
Well, you mean if somebody would write code like

```
for i in range(100000):
     self.set_status("hello")
```

that "caching" would cover that. But what about the slightly different:


```
for i in range(100000):
     self.set_status("hello " + i)
```

I think it would be good for stability if we checked that there were no more than 105 (or something) status_updates per run invocation. And I don't think this arbitrary check is the way to go. I don't want anyone starting to rely on it and it suddenly gets re-factored away.